swagger_to/style.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+"""
+Generates code for an Elm client.
+"""
+# pylint: disable=too-many-lines
+from typing import MutableSet, MutableMapping, List
+
+import swagger_to.intermediate
+import swagger_to.swagger
+import swagger_to
+
+
+def check_header(swagger: swagger_to.swagger) -> List[str]:
+    """
+    Checks whether the swagger header conforms to the style guide.
+
+    :param swagger: parsed swagger file
+    :return: the list of failed checks
+    """
+    errs = []  # type: List[str]
+
+    if swagger.name != swagger_to.snake_case(swagger.name):
+        errs.append("Swagger name {} is not snake case.".format(swagger.name))
```

Comment:
(Comment A) I'd put the offending value at the end for readability (`Name of the swagger specification is not in snake case: {}`). Please see also the messages below.